pilot/pkg/features/pilot.go

```diff
@@ -328,4 +328,12 @@ var (
 			"No need to configure this for root certificates issued via Istiod or web-PKI based root certificates. "+
 			"Use || between <trustdomain, endpoint> tuples. Use | as delimiter between trust domain and endpoint in "+
 			"each tuple. For example: foo|https://url/for/foo||bar|https://url/for/bar").Get()
+
+	EnableDefaultToSystemCAValidation = env.RegisterBoolVar("USE_SYSTEM_CA_FALLBACK", false,
```

Comment:
Too generic name and comment doesn't make sense? 

DESTINATION_RULE_DEFAULT_CERT ?